source/_footer.html.erb

```diff
@@ -0,0 +1,7 @@
+<footer>
+    <ul class="footer">
+        <li class="footer__link"><%= link_to '860-406-1021', 'tel:#{8604061021}' %></li>
```

Comment:
I would shy away from making your phone number public. Not entirely sure that's the safest, most conventional way to get in touch with you. I think email and Linkedin would suffice. Curious to hear your thoughts though!